### PR TITLE
Tag images with 'latest' and release tags

### PR DESCRIPTION
**Pull request by pierDipi**
This will help with generating the CSV file with the "latest" image SHAs

Here is how it will be used https://github.com/openshift-knative/serverless-operator/pull/2826

### CODE REVIEW

File: pkg/konfluxgen/konfluxgen.go [hunk 1]
```diff
@@ -181,7 +181,9 @@ func Generate(cfg Config) error {
 				Nudges:                        append(cfg.Nudges, cfg.NudgesFunc(c.ReleaseBuildConfiguration, ib)...),
 				Pipeline:                      pipeline,
 				AdditionalTektonCELExpression: cfg.AdditionalTektonCELExpressionFunc(c.ReleaseBuildConfiguration, ib),
+				Tags:                          []string{"latest"},
```
Comment (creydr) [inline pkg/konfluxgen/konfluxgen.go:184]: I probably miss something, but just to be sure here: as we use the same pipeline for on-push and on-pr, wouldn't we tag images from PRs also with `latest`?

Comment (pierDipi) [reply pkg/konfluxgen/konfluxgen.go:184]: no, the parameter is only added for `push` pipeline

Comment (pierDipi) [reply pkg/konfluxgen/konfluxgen.go:184]:
```
    {{{- if eq .Event "push" }}}
    - name: additional-tags
      value: [{{{ range $tag := .Tags }}} {{{ $tag }}}, {{{ end }}}]
    {{{- end }}}
```

Comment (pierDipi) [reply pkg/konfluxgen/konfluxgen.go:184]: we're injecting parameters for the pipeline run and we have one pipeline run per event (push and PR separate)

Comment (creydr) [reply pkg/konfluxgen/konfluxgen.go:184]: Ah nice :+1:

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**